subsys/net/ip/net_context.c

```diff
@@ -1164,22 +1121,22 @@ static int context_setup_udp_packet(struct net_context *context,
 	int ret = -EINVAL;
 	u16_t dst_port = 0U;
 
-	if (IS_ENABLED(CONFIG_NET_IPV4) &&
-	    net_context_get_family(context) == AF_INET) {
-		struct sockaddr_in *addr4 = (struct sockaddr_in *)dst_addr;
-
-		dst_port = addr4->sin_port;
-
-		ret = net_context_create_ipv4_new(context, pkt,
-						  NULL, &addr4->sin_addr);
-	} else if (IS_ENABLED(CONFIG_NET_IPV6) &&
-		   net_context_get_family(context) == AF_INET6) {
+	if (IS_ENABLED(CONFIG_NET_IPV6) &&
```

Comment:
This looks like unnecessary change as it just swaps ipv4 and ipv6 blocks.

Reply:
read the commit message. 75% of the context.c have this order: af_inet6 then af_inet, and while I made the net_pkt/net_context api changed, I reversed that order in some place. So I decided to move it back.

Reply:
There was just too many changes here that were not related -> more difficult to review. I have no issue if we reverse the if's, just put it to separate commit.